### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by illayarajah**
Completion of Homework 3

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            float height = returnFloat("Enter Your Height in M");
+            float bmi = calculateBMI(weight, height);
+            Console.WriteLine("Your BMI is " + bmi);
+            
+            
+
+
+
+
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: What's up with so many blank lines?

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            
+
+
+
+
+         
+        }
+
+
+        public static int returnInt(string message)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: PascalCase your method names

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        {
+            Console.WriteLine(message);
+            return Convert.ToInt32(Console.ReadLine());
+        }
+
+        public static string returnString(string message)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: You don't just return a string. You `PromptString`. Please rename. Don't use a word `return` in a name ever, it does not add any value.

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 1 above)
Comment (illayarajah) [reply Src/BootCamp.Chapter/Lesson3.cs]: Removed

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 2 above)
Comment (illayarajah) [reply Src/BootCamp.Chapter/Lesson3.cs]: Thank you, fixed as well.

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 3 above)
Comment (illayarajah) [reply Src/BootCamp.Chapter/Lesson3.cs]: done, thanks for reviewing the work.

**Comment by Almantask** [comment]
Looks good

**Pull request closed without merging**